src/sign.py

```diff
@@ -100,17 +112,27 @@ def _arguments() -> argparse.Namespace:
 def _get_payload_signer(args: argparse.Namespace) -> signing.Signer:
     if args.method == "private-key":
         _check_private_key_options(args)
-        return key.ECKeySigner.from_path(private_key_path=args.key_path)
+        payload_signer = key.ECKeySigner.from_path(
+            private_key_path=args.key_path
+        )
+        return in_toto_signature.IntotoSigner(payload_signer)
     elif args.method == "pki":
         _check_pki_options(args)
-        return pki.PKISigner.from_path(
+        payload_signer = pki.PKISigner.from_path(
             args.key_path, args.signing_cert_path, args.cert_chain_path
         )
+        return in_toto_signature.IntotoSigner(payload_signer)
+    elif args.method == "sigstore":
+        return sigstore.SigstoreDSSESigner(
+            use_ambient_credentials=args.use_ambient_credentials
+        )
     elif args.method == "skip":
```

Comment:
@mihaimaruseac Do you know the need to support this `skip` signing option? A cursory search did not reveal where it is used/needed.

Reply:
I think it got added to see just the hashing part, for debugging? I don't think we need to support it for production.

Reply:
Ok, we can leave it in for now and remove when we're closer to production.